File: BaekJoon Online Judge/C++/Step01-I:O and Arithmetic/10430.cpp
//
//  10430.cpp
//  Step1-I/O-and-Arithmetic
//
//  Created by 김예빈 on 2019. 8. 28..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    int a, b, c;
    
    cin >> a >> b >> c;
    
    cout << (a + b) % c << endl;
    cout << (a % c + b % c) % c << endl;
    cout << (a * b) % c << endl;
    cout << (a % c * b % c) % c << endl;
    
    return 0;
}

File: BaekJoon Online Judge/Swift/Step01-I:O and Arithmetic/10430.swift
//
//  10430.swift
//  Step01-I/O and Arithmetic
//
//  Created by Yebin Kim on 2019/10/27.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

var input = readLine()

if let input = input {
    let arr = input.components(separatedBy: " ")
    let A = Int(arr[0])!
    let B = Int(arr[1])!
    let C = Int(arr[2])!
    
    let ans1 = (A + B) % C
    let ans2 = (A % C + B % C) % C
    let ans3 = (A * B) % C
    let ans4 = (A % C * B % C) % C
    
    print(ans1)
    print(ans2)
    print(ans3)
    print(ans4)
}
